tests/server/server-executeCommand-002.phpt

```diff
@@ -0,0 +1,78 @@
+--TEST--
+MongoDB\Driver\Server::executeCommand() takes a read preference
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; NEEDS("REPLICASET"); ?>
```

Comment:
I think most IDEs allow you to hover over a filename and "click it" to goto the file.
In vim it is `gf` ("goto file"), or `gF` (if the string is in the format of `filename:linenumber`, then goto file on line `linenumber`)